.github/workflows/build.yml

```
@@ -46,9 +46,12 @@ jobs:
 
       - name: Set up Java  ${{ env.JAVA_VERSION }}
         if: env.NODE_VERSION == '14'
-        uses: actions/setup-java@v1.4.3
+        uses: actions/setup-java@v2.0.0
         with:
           java-version: ${{ env.JAVA_VERSION }}
+          distribution: 'adopt'
+          java-package: jdk
+          check-latest: false
```

```suggestion
          check-latest: true
```

If you want to test on the latest version of Java 11 all the time you can set `check-latest` to `true` with this suggestion.
The tradeoff is that if the latest version of Java 11 is not in the Action Runner environment, it will download the latest version, which is slower than grabbing it from the environment or cache.

I'll let the Renovate team decide what they like best.

---

i think using cached version is ok, as we only need it for testing.